coral-component-shell/examples/index.html

```diff
@@ -117,6 +117,14 @@ <h2 class="coral-Heading coral-Heading--4 u-coral-margin">Pulse notifications</h
               </coral-shell-solutions>
             </coral-shell-solutionswitcher>
           </coral-shell-menu>
+          <coral-shell-menu id="menu_selectlistswitcher" placement="right" from="top">
+            <coral-shell-selectlistswitcher>
+              <coral-shell-switcherlist-item href="http://www.adobe.com/go/aem6_5_experiencecloud">Experience Cloud</coral-shell-switcherlist-item>
+              <coral-shell-switcherlist-item href="http://www.adobe.com/go/aem6_5_advertisingcloud">Advertising Cloud</coral-shell-switcherlist-item>
+              <coral-shell-switcherlist-item href="http://www.adobe.com/go/aem6_5_analytics">Analytics</coral-shell-switcherlist-item>
+              <coral-shell-switcherlist-item href="http://www.adobe.com/go/aem6_5_audiencemanager">Audience Manager</coral-shell-switcherlist-item>
```

Comment:
I think there is a problem within the example: `coral-shell-switcherlist-item` elements should be `coral-shell-selectlistswitcher-item`.

Reply:
I have made the necessary changes @majornista